addon/components/nrg-sidebar-menu-item/component.js

```diff
@@ -19,4 +19,10 @@ export default Component.extend({
   _linkToItemClass: computed('isParentGroup', function() {
     return this.get('isParentGroup') ? 'parent-route': 'item';
   }),
+
+  actions: {
+    clickedLink(item) {
+      this.clicked(item);
+    },
+  },
```

Comment:
If you use the handlebars suggestion in the template, you will not need this. Well should not at least.